### PR TITLE
fix travis (flake8)

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

**Comment by lepistone** [comment]
:+1:

**Comment by pedrobaeza** [comment]
```
./tr_barcode/tr_barcode.py:32:5: F821 undefined name '_logger'
```

File: tr_barcode/tr_barcode.py [hunk 1]
```diff
@@ -33,6 +32,9 @@
     _logger.warning("ERROR IMPORTING REPORT LAB")
```
Comment (yvaucher) [inline tr_barcode/tr_barcode.py:33]: Erm here is a case for which this rule is wrong unless it is possible to define logger just before `try .. except`

Comment (pedrobaeza) [inline tr_barcode/tr_barcode.py:33]: Put it just right after except (duplicating it)

Comment (yvaucher) [inline tr_barcode/tr_barcode.py:33]: Or we could ignore E402 in such case

Comment (lepistone) [inline tr_barcode/tr_barcode.py:33]: no need, the try-except blocks are ignored already. Just put `_logger` after the last top-level import.

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**